ygnmi/bench_test.go

```diff
@@ -28,24 +29,31 @@ import (
 	gpb "github.com/openconfig/gnmi/proto/gnmi"
 )
 
+// subClient will send count notifications, then return EOF
+// if count > len(resp), the last notification is sent repeatedly
```

Comment:
could you add a period after the first sentence? I thought it was a full sentence and was wondering why it returns EOF if count > len(resp)